src/api.v2/helpers/pipeline/handleQCResponse.js

```diff
@@ -60,7 +60,7 @@ const updateProcessingConfigWithQCStep = async (taskName, experimentId, output,
   // TODO the processing config validation was not being enforced because the 'anyOf' requirement
   // was not being correctly applied. This needs to be refactored together with the
   // pipeline and ideally while unifying the qc & gem2s responses.
-  // await validateRequest(output, 'ProcessingConfigBodies.v1.yaml');
+  await validateRequest(output, 'ProcessingConfigBodies.v2.yaml');
```

Comment:
this will probably cause the requests to fail to so I don't think it's a good idea to uncomment this as part of such a big PR because it will make debugging harder imo

Reply:
I tested it and it didn't fail the validation